sdk/typescript/src/types/events.ts

```diff
@@ -54,3 +55,40 @@ export type SuiEvent =
     | { newObject: NewObjectEvent }
     | { epochChange: bigint }
     | { checkpoint: bigint };
+
+export type MoveEventField = {
+    path: string,
+    value: SuiJsonValue
+}
+
+export type EventType =
+    | "MoveEvent"
+    | "Publish"
+    | "TransferObject"
+    | "DeleteObject"
+    | "NewObject"
+    | "EpochChange"
+    | "Checkpoint";
+
+// mirrors sui_json_rpc_types::SuiEventFilter
+export type SuiEventFilter =
+    | { "package" : ObjectId }
+    | { "module" : string }
+    | { "moveEventType" : string }
+    | { "moveEventField" : MoveEventField }
+    | { "senderAddress" : SuiAddress }
+    | { "eventType" : EventType }
+    | { "All" : SuiEventFilter[] }
+    | { "Any" : SuiEventFilter[] }
+    | { "And" : [SuiEventFilter, SuiEventFilter] }
+    | { "Or" : [SuiEventFilter, SuiEventFilter] };
```

Comment:
the capitalization of these events is not consistent. Could you test out all these cases?

Reply:
good catch. they're all capitalized now.